gmgc_mapper_plotter/GMGC_Mapper_plotter.ipynb

```diff
@@ -0,0 +1,222 @@
+      "text/plain": [
+       "<Figure size 576x432 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "graf_4 = plt.figure(figsize=(8, 6))\n",
+    "ax = graf_4.gca()\n",
+    "ax.bar(x,y, color='powderblue')\n",
+    "#ax.set_xlabel(\"\", labelpad=10, weight='bold', size=12)\n",
+    "ax.set_ylabel(\"Counts\", labelpad=10, weight='bold', size=12)\n",
+    "ax.set_title(\"GMGC mapper\", weight='bold', size=14)\n",
+    "for i  in (range(len(y))):\n",
+    "    ax.text(i , y[i]+3 , str(y[i])+'('+str(round(int(y[i])*100/int(t),1))+'%)', horizontalalignment='center', color='teal', fontweight='bold')\n",
```

Comment:
The `str(y[i]) + '('+str...` bit can probably be replaced by a single f-string: `f"{y[i]} ({y[i]/t:.1%})"` 

See https://www.python.org/dev/peps/pep-0498/